client/src/components/utils/Login/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Button from '../Button/index';
+import background from '../../../assets/images/login.background.png';
+import waiter from '../../../assets/images/waiter.png';
+import './style.css';
+
+export default class Login extends React.Component {
+  state = {
```

Comment:
The state of the table number and the secret should be in the app (root component) to be accessible to all components